src/components/EditorTasks/EditorTasks.js

```diff
@@ -0,0 +1,63 @@
+import React, { useCallback, useMemo } from 'react';
+import { Button } from '../../common/modules/Button/Button';
+import { INFO, SUCCESS, WARNING } from '../../common/constants/constantsColorButton/constantsColorButton';
+import { Icon } from '../../common/modules/Icons/Icons';
+import { COMPLETED, IN_PROGRESS, REJECTED } from '../../common/constants/constantsTasks/constantsTasks';
+import { completedTask, rejectedTask, todoTask } from '../../store/tasks/actionsTasks';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectCheckedTask, selectMarkedTask } from '../../store/selectedEntity/selectorSelects';
+import { setSelectTask } from '../../store/selectedEntity/actionsSelects';
+import s from './EditTasks.module.css';
+
+export const EditorTasks = () => {
+  const dispatch = useDispatch();
+  const markTasksId = useSelector(selectMarkedTask);
+  const selectedTasks = useSelector(selectCheckedTask);
+  const isVisebled = Object.values(selectedTasks).filter(item => item).length;
+
+  const handleRejectedTask = useCallback(() => {
+    markTasksId.forEach((id) => {
+      dispatch(rejectedTask(id));
+      dispatch(setSelectTask({[id]: false}));
+    });
+  }, [dispatch, markTasksId]);
+
+  const handleCompletedTask = useCallback(() => {
+    markTasksId.forEach((id) => {
+      dispatch(completedTask(id));
+      dispatch(setSelectTask({[id]: false}));
+    });
+  }, [dispatch, markTasksId]);
+
+  const handleTodoTask = useCallback(() => {
+    markTasksId.forEach((id) => {
+      dispatch(todoTask(id));
+      dispatch(setSelectTask({[id]: false}));
+    });
+  }, [dispatch, markTasksId]);
+
+  const handleEditorPosition = useMemo(() => {
+    return isVisebled && s.collapsed;
```

Comment:
можно упростить в одну строку 
```suggestion
    const handleEditorPosition = useMemo(() => isVisebled && s.collapsed, [selectedTasks]);
```

Reply:
Спасибо, полностью согласен